src/mdesktopentry.cpp

```diff
@@ -47,12 +48,15 @@ const QString CategoriesKey("Categories");
 const QString StartupNotifyKey("StartupNotify");
 const QString StartupWMClassKey("StartupWMClass");
 const QString URLKey("URL");
-const QString LogicalIdKey("X-MeeGo-Logical-Id");
-const QString TranslationCatalogKey("X-MeeGo-Translation-Catalog");
+const QString LogicalIdTemplate("X-Amber-%1-Logical-Id");
```

Comment:
Pondering two things:
- Would it be cleaner if %1 was the last part of the template. 
- What was logical here anyways? Quickly didn't spot the word that much used in Qt side. "Translation-Id" maybe?

Reply:
I agree with these. I have no idea where that Logical word comes from, Translation sounds much better. I'll change these.